webpack.config.js

```diff
@@ -59,6 +59,14 @@ module.exports = (env, argv) => { // eslint-disable-line no-undef
             }),
             new CleanObsoleteChunksPlugin(),
         ],
+        resolve: {
+            alias: {
+                'fos-jsrouting': path.resolve(
```

Comment:
🙈

Reply:
Works perfectly :stuck_out_tongue: Although I would still prefer to have a NPM package provided by them...

Reply:
There seems to be an issue for that already. Keeping an eye on it: https://github.com/FriendsOfSymfony/FOSJsRoutingBundle/issues/342